src/App.tsx

```diff
@@ -1,24 +1,179 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import {
+  FC,
+  useState,
+  useEffect,
+  useMemo,
+  useCallback,
+} from 'react';
+import { TodoService } from './api/todos';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { TodoStatus } from './types/TodoStatus';
+import { filterTodos } from './utils/helpers';
 
-export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+import { AddingForm } from './components/AddingForm';
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { NotificationError } from './components/NotificationError';
+import { Errors } from './types/Errors';
+
+const USER_ID = 6922;
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [todoStatus, setTodoStatus] = useState<TodoStatus>(TodoStatus.ALL);
+  const [loadingTodosIds, setLoadingTodosIds] = useState<number[]>([]);
+  const [isInputDisabled, setIsInputDisabled] = useState<boolean>(false);
+  const [error, setError] = useState<Errors>(Errors.NONE);
+
+  const loadTodos = useCallback(async () => {
+    try {
+      const loadedTodos = await TodoService.getByUserId(USER_ID);
+
+      setTodos(loadedTodos);
+    } catch (err) {
+      setError(Errors.LOAD);
+    }
+  }, []);
```

Comment:
you don't pass loadTodos to child components, so what's the purpose of this hook here?